package.json

```diff
@@ -121,7 +123,7 @@
     "go-ipfs": "0.6.0",
     "http-proxy": "^1.18.1",
     "http-server": "^0.12.3",
-    "ipfs": "^0.48.1",
+    "ipfs": "0.50.2",
```

Comment:
is this intended in this PR? Just want to make sure we test everything so we don't problems from upgrading ipfs :)

Reply:
Good catch! js-ipfs is used only in e2e tests. 

Not related to this PR, but I noticed we were not using hard-coded version so fixed it and bumped to the latest version to ensure we are green with the latest version.